micrometer-core/src/main/java/io/micrometer/core/instrument/filter/FilterProperties.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.filter;
+
+import io.micrometer.core.instrument.ImmutableTag;
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.Tag;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class FilterProperties {
+    public static final String INCLUDE = "include";
+    public static final String EXCLUDE = "exclude";
+
+    private String defaultCombinedBucketName = "other";
+    private String defaultFilter = INCLUDE;
+    private Map<String, String> filter = new HashMap<>();
+    private Map<String, String> combine = new HashMap<>();
+
+    public String getDefaultFilter() {
+        return defaultFilter;
+    }
+
+    public void setDefaultFilter(String defaultFilter) {
+        this.defaultFilter = defaultFilter;
+    }
+
+    public void setDefaultCombinedBucketName(String defaultCombinedBucketName) {
+        this.defaultCombinedBucketName = defaultCombinedBucketName;
+    }
+
+    public String getDefaultCombinedBucketName() {
+        return defaultCombinedBucketName;
+    }
+
+    public Map<String, String> getFilter() {
+        return filter;
+    }
+
+    public Map<String, String> getCombine() {
+        return combine;
+    }
+
+    public String findMostSpecificRule(String name, Map<String,String> map, String defaultVal) {
```

Comment:
Current implementation only finds the most specific rule. I can add in wildcard support and multiple rules in a subsequent PR.